mlcvs/utils/data.py

```diff
@@ -33,6 +33,46 @@ def closest_idx_torch(array, value):
         else:
             return pos-1
 
+# evaluation of tprime from simulation time and logweights
+def tprime_evaluation(t, logweights = None):
+    """
+    Estimate the accelerated time if a set of (log)weights is given
+
+    Parameters
+    ----------
+    t : array-like, 
+        unbias time series,    
+    logweights : array-like,optional
+        logweights to evaluate rescaled time as dt' = dt*exp(logweights)
+    """
+
+    # rescale time with log-weights if given
+    if logweights is not None:
+        # compute time increment in simulation time t
+        dt = np.round(t[1]-t[0],3)
+        # sanitize logweights
+        logweights = torch.Tensor(logweights)
+        # when the bias is not deposited the value of bias potential is minimum 
+        logweights -= torch.max(logweights)
+        # bug: exp(logweights/lognorm) != exp(logweights)/norm, where norm is sum_i beta V_i
+        # are we changing the temperature? 
+        """ possibilities:
+            1) logweights /= torch.min(logweights) -> logweights belong to [0,1] 
+            2) pass beta as an argument, then logweights *= beta
+            3) tprime = dt * torch.cumsum( torch.exp( torch.logsumexp(logweights,0) ) ,0)
+            4) tprime = dt *torch.exp ( torch.log (torch.cumsum (torch.exp(logweights) ) ) )
+        """
```

Comment:
do you mean instead of doing logweights -= max(logweights) ? 
I can try, but I think it will not change the estimates relative to the time correlation functions.